packages/suite/src/views/dashboard/components/AssetsView/components/AssetCard.tsx

```diff
@@ -147,7 +150,7 @@ export const AssetCard = ({
                         <FiatAmount>
                             <FiatHeader
                                 size="medium"
-                                portfolioValue={cryptoValue}
+                                portfolioValue={fiatAmount ?? '0'}
```

Comment:
The hook should be a part of the `FiatHeader` component :) All amounts passed in all the value-related components are in crypto, not fiat. 

Also the `FiatValue` component is able to provide the fiat value to its [children](https://github.com/trezor/trezor-suite/blob/ab34249019dc721205edea90c1b277afccc8786c/packages/suite/src/components/suite/Ticker/TrendTicker.tsx#L48), which I agree is not the best but then you do not need the hook :)

Reply:
In my opinion component `FiatValue` is too heavy and it makes sense to have extracted logic somewhere outside. Also I agree with you that `FiatHeader` should accept crypto and make all the magic inside. I will explore it more.